src/app.ts

```diff
@@ -190,19 +191,7 @@ const getPublicAdministrationsHandler: RequestHandler = async (
     const foundPublicAdministrations = await findPublicAdministrationsByName(
       req.query.search
     );
-    res.json(
-      foundPublicAdministrations.map(foundPublicAdministration => {
-        return {
-          ...foundPublicAdministration,
-          links: [
-            {
-              href: `/public-administrations/${foundPublicAdministration.ipaCode}`,
-              rel: "self"
-            }
-          ]
-        };
-      })
-    );
+    res.json(foundPublicAdministrations);
```

Comment:
this looks like an express controller. as a such it should follow other controllers convention and semantic (like: having it in a separate standalone module like [ProfileController](https://github.com/teamdigitale/io-onboarding-pa-api/blob/master/src/controllers/profileController.ts) here in the same project)

Reply:
moreover the return type of this controller is untyped unlike the others. is there any reason on why this is hardcoded into app.ts ?